documentation/developer-guide/modules/tooling-guide/pages/bamm-cli.adoc

```diff
@@ -140,6 +140,214 @@ The full command would result in:
+the whole procedure is applied recursively to the complex type's properties
+
+
+==== Mapping of Aspect's operations
+
+If the Aspect also has a non-empty list of operations defined, like the one in the following example:
+
+----
+:AspectWithOperation a bamm:Aspect ;
+   bamm:properties ( ) ;
+   bamm:operations ( :testOperation ) .
+
+:testOperation a bamm:Operation ;
+   bamm:input ( :input ) ; <1>
+   bamm:output :output . <2>
+
+:output a bamm:Property ;
+   bamm:characteristic bamm-c:Text . <3>
+
+:input a bamm:Property ;
+   bamm:characteristic bamm-c:Text . <4>
+----
+
+then additional entries are added to the generated OpenAPI specification.
+First, there is an additional entry in the `paths` section of the specification: `/{tenant-id}/aspect-with-operation/*operations*`.
+The available operations are then added to the `components/schemas` part:
+
+[source,JSON]
+----
+{
+ "Operation" : {
+    "allOf" : [ {
+      "$ref" : "#/components/schemas/JsonRpc"
+    }, {
+      "properties" : {
+        "params" : {
+          "type" : "object",
+          "required" : [ "input" ], // <1>
+          "properties" : {
+            "input" : { // <1>
+              "$ref" : "#/components/schemas/urn_bamm_io.openmanufacturing_characteristic_2.0.0_Text" // <3>
+            }
+          }
+        },
+        "method" : {
+          "type" : "string",
+          "description" : "The method name",
+          "example" : "testOperation"
+        }
+      }
+    } ]
+  },
+ "OperationResponse" : {
+    "allOf" : [ {
+      "$ref" : "#/components/schemas/JsonRpc"
+    }, {
+      "properties" : {
+        "result" : {
+          "type" : "object",
+          "required" : [ "output" ], // <2>
+          "properties" : {
+            "output" : { // <2>
+              "$ref" : "#/components/schemas/urn_bamm_io.openmanufacturing_characteristic_2.0.0_Text" // <4>
+            }
+          }
+        }
+      }
+    } ]
+  }
+}
+----
+<1> the names of the input
+<2> and output parameters are reflected in the properties generated for the request/response objects
+<3> the characteristics are generated
+<4> and referenced as described in the point 6 of the section "Mapping of Aspect and its properties"
+
+As usual, corresponding entries referencing the definitions above are added to the `requestBodies` and `responses` sections (examples omitted for simplicity).
+For technical reasons, there may be a slight variation in the generated JSON depending on whether the aspect has one or more operations defined.
+
+==== Mapping of Collections
+
+There are some additional JSON entries generated for complex types related to various types of collections to facilitate
+access to the individual elements of these collections via paging. As these entries are rather of static character
+without direct references to any aspect elements, it suffices here to give a short overview about which kind of paging
```

Comment:
Although not technically part of the mapping of model elements to OpenAPI, this is not documented anywhere else. So IMO, it would make sense to show small examples (or extracts) of how the various paging strategies are represented in the OpenAPI.